src/rf/js/src/home/components/processing-block.js

```diff
@@ -17,7 +17,7 @@ var LayerStatusComponent = React.createBackboneClass({
             chunkClass = this.pendingClass,
             mosaicClass = this.pendingClass,
             completeClass = this.pendingClass,
-            actionLinks = (<a className="text-danger">Cancel</a>),
+            actionLinks = (<a href="javascript:;" onClick={this.deleteLayer} className="text-danger">Cancel</a>),
```

Comment:
The CSS is already setup so you shouldn't need to do this. https://github.com/azavea/raster-foundry/blob/develop/src/rf/sass/components/_processing-block.scss#L100